contracts/eosio.system/src/producer_pay.cpp

```diff
@@ -71,12 +71,16 @@ namespace eosiosystem {
       const auto usecs_since_last_fill = (ct - _gstate.last_pervote_bucket_fill).count();
 
       if( usecs_since_last_fill > 0 && _gstate.last_pervote_bucket_fill > time_point() ) {
+         const auto unstake_time = std::min(current_time_point(), gbm_final_time);
+         const int64_t delta_time_usec = (gbm_final_time - unstake_time).count();
          auto new_tokens = static_cast<int64_t>( (continuous_rate * double(token_supply.amount) * double(usecs_since_last_fill)) / double(useconds_per_year) );
-         // needs to be 1/2 Savings, 1/6 Producers, 1/6 Voters, 1/6 Genesis Block Member
-         auto to_voters        = new_tokens / 6;
+         // needs to be 1/2 Savings, 1/5 Voters, GBM receives a linearly deflating share over three years
+         auto to_voters        = new_tokens / 5;
          auto to_per_block_pay = to_voters;
-         auto to_gbm           = to_voters;
+         auto to_savings       = new_tokens - (to_voters + to_per_block_pay);
+         auto to_gbm           = to_voters * 2 * (delta_time_usec / double(useconds_in_gbm_period));
          auto to_savings       = new_tokens - (to_voters + to_per_block_pay + to_gbm);
```

Comment:
hmm I think this line needs to be deleted. The assignment in line 80 looks like the correct value

Reply:
Sure!, I believe this is a residual of the merge :man_facepalming:. Thanks.